pactflow/client/ai.ts

```diff
@@ -200,6 +245,24 @@ export const GenerationInputSchema = z.object({
   ),
 });
 
+export const GenerationToolInputSchema = z.object({
+  body: GenerationInputSchema.describe(
+    "The body of the request to be used for Pact test generation."
+  ),
+  remoteOpenAPIDocument: RemoteOpenAPIDocumentSchema.optional().describe(
+    "The remote OpenAPI document to use for the review in case openapi document is not provided. If provided do not include the document field under openapi."
+  ),
+});
```

Comment:
This feels somewhat incorrect as a type. From what I understand, the input should _either_ be a local file _or_ a remote file. This would imply that you should use an enum or some `A | B` type. What you have you imply that you can have both a `body` and a `remoteOpenAPIDocument`.